core/src/main/java/io/javaoperatorsdk/jenvtest/KubeAPIServerConfigBuilder.java

```diff
@@ -58,6 +61,28 @@ public KubeAPIServerConfig build() {
         this.apiServerVersion = apiServerVersionEnvVar;
       }
     }
-    return new KubeAPIServerConfig(jenvtestDir, apiServerVersion, offlineMode);
+    return new KubeAPIServerConfig(jenvtestDir, apiServerVersion, offlineMode, apiServerFlags);
+  }
+
+  public void withApiServerFlags(List<String> flags) {
+    apiServerFlags.addAll(flags);
+  }
+
+  public void withApiServerFlag(String key, String value) {
+    checkKeyPrefix(key);
+    apiServerFlags.add(key);
+    apiServerFlags.add(value);
+  }
+
+  public void withApiServerFlag(String key) {
+    checkKeyPrefix(key);
+    apiServerFlags.add(key);
+  }
+
+  private void checkKeyPrefix(String key) {
```

Comment:
Maybe it'd be better to issue a warning and add the prefix automatically if it's missing to be more user-friendly?

Reply:
actually yes, was thinking to ask for the flag without the flag

Reply:
it's a problem with the list of flags. I will merge this simple form, and we can do impovements in a subsequent PR.